src/java.base/share/classes/javax/security/auth/spi/LoginModule.java

```diff
@@ -26,9 +26,8 @@
 package javax.security.auth.spi;
 
 import javax.security.auth.Subject;
-import javax.security.auth.AuthPermission;
-import javax.security.auth.callback.*;
-import javax.security.auth.login.*;
+import javax.security.auth.callback.CallbackHandler;
```

Comment:
Out of curiosity, did IJ point this out somehow?

Reply:
Yes. IJ complained about unused imports and offered to optimize them for me.

Reply:
Ah, so this is where it asked if you wanted to optimize.